### ISSUE TITLE
Illuminate/foundation is an orphan?

### ISSUE THREAD

**Issue by IgorDePaula** [laravel/framework#24402]
- Laravel Version: 5.6
- PHP Version: 7.2.5
- Database Driver & Version: mysql 5.7

### Description:
I need helpers from illuminate/foundation for a package test, but its abandoned. 

### Steps To Reproduce:
Try composer install with illuminate/database:5.6

**Comment by staudenmeir** [comment]
You mean `illuminate/foundation:5.6`?

**Comment by devcircus** [comment]
Foundation exists solely to glue the framework to the components. It doesn't really make sense as a standalone package. If there are some methods you like to use from foundation, just grab the ones you need or even throw them into a package. 

Also, Illuminate database doesn't require foundation:
```
    "require": {
        "php": "^7.1.3",
        "illuminate/container": "5.7.*",
        "illuminate/contracts": "5.7.*",
        "illuminate/support": "5.7.*"
    },
```
It's been discussed [here](https://github.com/laravel/framework/issues/6323)

**Comment by IgorDePaula** [comment]
I need use the config_path helper, by example, how I test using this helper?

**Comment by IgorDePaula** [comment]
Do you recommend me the orchestral/testbench package?

**Comment by BafS** [comment]
@devcircus Yes it's the glue but some importants pieces of code are inside, for example the caching system for the container is in Application.php, in Foundation.
